tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -174,8 +174,12 @@ Spawn Notebook With Arguments  # robocop: disable
          Click Element  xpath://span[@id='jupyterhub-logo']
       ELSE
          Click Element  xpath://span[@id='jupyterhub-logo']
+         Sleep  ${retries_delay}
       END
    END
+   IF ${spawn_fail} == True
```

Comment:
There's only one whitespace between the IF and ${spawn_fail}, so the test are currently failing:
```
1) No keyword with name 'IF ${spawn_fail} == True' found.
```

@kpouget please update it to be two whitespaces or more, `IF  ${spawn_fail} == True`

Reply:
fixed (the `robot` language is hard 🙃)

Reply:
I would say annoying more than hard :p